types.go

```diff
@@ -1527,25 +1527,25 @@ func (rr *EUI64) copy() RR           { return &EUI64{*rr.Hdr.copyHeader(), rr.Ad
 func (rr *EUI64) String() string     { return rr.Hdr.String() + euiToString(rr.Address, 64) }
 func (rr *EUI64) len() int           { return rr.Hdr.len() + 8 }
 
-// Support in incomplete - just handle it as unknown record
-/*
 type CAA struct {
 	Hdr   RR_Header
 	Flag  uint8
 	Tag   string
-	Value string `dns:"octet"`
+	Value string `dns:"hex"`
```

Comment:
I originally just decoded this directly to a string but this noticed in section 4.1 of [RFC6844](https://tools.ietf.org/html/rfc6844)

```
Value:  A sequence of octets representing the property value.
   Property values are encoded as binary values and MAY employ sub-
   formats.
```

90% of the records I've found have binary encoded ASCII content in `Value` but a few (e.g. for domains `madtech.nl, codewiz.org, www.isoc.lu` etc) contain non-ASCII binary data. My thinking was that since the RFC explicitly defines it to be binary encoded (and possibly containing sub-formats) it would be better to just return a `hex` string and let the user decide what to do with it based on the `Tag` contents.

I'd be happy to switch this back to a `string` (or perhaps a `byte` slice?) if you think that would be more appropriate.